src/components/market/constants.js

```diff
@@ -0,0 +1,184 @@
+export const PRODUCT = {
+    ACTIONS: {
+        INFO_CIRCLE: 'info-circle',
+        SHOPPING: 'shopping',
+        SHOPPING_CART: 'shopping-cart',
+    },
+    CARD_IMAGE: 'card-image',
+    DEFAULT_DESCRIPTION: 'This is the product description',
+    MODAL_INFO: {
+        BUY: 'Buy Now',
+        CANCEL: 'Cancel',
+        COST: 'Cost:',
+        EMAIL: 'Email:',
+        LOCATION: 'Location:',
+        PHONE: 'Phone #:',
+        PRODUCERS_NAME: `Producer's Name:`,
+        QUANTITY: 'Available Quantity:',
+    },
+    TAG_COLOR: 'gold',
+};
+
+export const EXAMPLE_PRODUCTS = [
```

Comment:
get the products from the backend not from this constants file

Reply:
ok